iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/Program.cs

```diff
@@ -77,7 +77,7 @@ private static ILogger InitializeConsoleDebugLogger()
             {
                 builder
                 .AddFilter(level => level >= LogLevel.Debug)
-                .AddConsole(options =>
+                .AddSystemdConsole(options =>
```

Comment:
Can you say more about this change?

Reply:
https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/5.0/obsolete-consoleloggeroptions-properties

ConsoleLoggerOptions properties have been deprecated, it's recommended to use ConsoleFormatterOptions properties instead.

Reply:
This was the only warning that started popping up on the Device samples that I had to fix. back to 0.